packages/expo-image/android/src/main/java/expo/modules/image/records/SourceMap.kt

```diff
@@ -46,10 +47,18 @@ data class SourceMap(
       parsedUri = computeUri(context)
     }
 
-    if (isContentUrl() || isDataUrl() || isBlurhash()) {
+    if (isContentUrl() || isDataUrl()) {
       return GlideRawModel(uri)
     }
 
+    if (isBlurhash()) {
+      return GlideBlurhashModel(
+        parsedUri!!,
```

Comment:
Sorry for `!!` 😛 
When the `isBlurhash` returns true the `parsedUri` cannot be null, but the compiler doesn't know that :/